Monika After Story/game/script-ch30.rpy

```diff
@@ -485,6 +485,26 @@ init python:
             rain weather to use, or None if we dont want to change weather
         """
         if mas_isMoniNormal(higher=True):
+            if mas_isSpring() and random.randint(1,10) > 5:
```

Comment:
can you change this so that we just roll randint once and then do all comparisons based on that chance?
also add comments to more clearly say what the chances are
could look something like:
```python
chance  = random.randint(1,100)
if spring and chance <= 50: 
   # 50% chance rain in the spring
   if chance <= 20:
      # 20% chance thunder (40% of the 50% chance):
      return thunder

   return rain
elif summer and chance <= 10:
   # 10% chance rain in the summer
   if chance <= 6:
      # 6% chance thunder (60% of 10%)
      return thunder

   return rain
elif fall and chance <= 30:
   # 30% chance rain in fall
   if chance <= 12:
      # 12% chance thunder (40% of 30%)
      return thunder
   return rain
```